schema/deploy/util_functions/get_form_status.sql

```diff
@@ -10,6 +10,14 @@ $function$
 
   select
     case
+        -- return not started for empty project form
+      when $2 = 'project'
+        and (fc.change_status = 'pending')
+        and (select cif.form_change_is_pristine((select row(form_change.*)::cif.form_change from cif.form_change where id=fc.id)) is distinct from true)
+        and (
+          ((select new_form_data from cif.form_change where id=fc.id) is null)
+          or ((select new_form_data from cif.form_change where id=fc.id) = jsonb_build_object()))
+        then 'Not Started'
```

Comment:
We're going to need tests for this function, but that can be in a tech debt card

Reply:
I was going to leave it for the card about refactoring the statuses, happy to start one out this morning though if we would rather get one in now